docs/release/V040_RELEASE_CHECKLIST.md

```
@@ -0,0 +1,902 @@
+    python -c "from omnibase_core.nodes import NodeCompute, NodeReducer, NodeOrchestrator, NodeEffect; print('Import verification OK')"
+    ```
+  - Expected: All commands succeed, version prints "0.4.0", import prints "Import verification OK"
+  - Evidence: Command outputs and hash comparison
+
+- [ ] **Git tag created**
+  - Tag: v0.4.0
+  - Command: `git tag -l v0.4.0`
+  - Evidence: Tag exists in repository
+
+- [ ] **GitHub release published**
+  - Release page: https://github.com/OmniNode-ai/omnibase_core/releases/tag/v0.4.0
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)